### DISCUSSION TITLE
PRO Badge doesn't appear on profile page

### DISCUSSION THREAD

**Discussion by EDM115** [community/community#31936] [Mobile] [answered]
As said [here](https://github.com/orgs/community/discussions/2807#discussioncomment-139847)  
  
Example :  
  
![on desktop version](https://telegra.ph/file/7a418ba289410e6e27f50.png)  
  
![on mobile version](https://telegra.ph/file/3e14c66a2601bff7a68c8.png)

**Comment by melvincwng** [comment, marked as answer]
@EDM115, unfortunately certain features of your GitHub Profile are only available in desktop view and not in mobile view.

Hope this clarifies.

**Comment by nurahmed123** [comment]
![image](https://user-images.githubusercontent.com/84973115/189472865-701e7032-1b1d-4ec4-881e-5585a0f54210.png)
I refreshed the page too many times but it did not appear the pro badge on the profile

**Reply by EDM115** [reply]
The pro badge appears only if you have subscribed to the GitHub Pro plan

**Comment by nurahmed123** [comment]
yes. I have already subscribed to the GitHub student plan a few months ago,
it appeared on my profile then. but suddenly I noticed that it's not
showing on my profile

On Sat, Sep 10, 2022 at 2:48 PM EDM 1.15 ***@***.***> wrote:

> The pro badge appears only if you have subscribed to the GitHub Pro plan
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/31936#discussioncomment-3613629>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUIJMO6I5WFLB627KJJNQS3V5RDOVANCNFSM6AAAAAAQELRYAE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 

Nur Ahmed

Web developer

***@***.***

Instagram <https://www.instagram.com/06nurahmed/> | Facebook
<https://www.facebook.com/EEE-project-solution-lab-107106394047092> |
YouTube <https://www.youtube.com/c/BongProgramiz>
